<launch>
  
  <node pkg="nao_behavior_tree" type="nao_leds" name="nao_leds" output="screen">
    <param name="NAO_IP" value="192.168.1.19"/>
    <param name="NAO_PORT" value="9559"/>
  </node>
  
</launch>
